### ISSUE TITLE
As an attendee, I might find it useful to list my pronouns somewhere

### ISSUE THREAD

**Issue by migetman9** [magfest/ubersystem#2547]
Brought to me was a suggestion that an attendee may want to specify their pronouns. I think this idea comes with a lot of discussion of what is right for the attendee and what is right for the system.  

Some preliminary thoughts are: Is it a pre-selected list? Do we let people add their own like affiliates where they can see other used pronouns, or just an other box for someone to write whatever. Does this pose any actual functionality to admins interacting with attendees?

**Comment by earl7399** [comment]
Does MAGFest/RAMS even use pronouns? Other than possibly "you" I cannot
think of any.

--
Robert Earl <r.a.earl@gmail.com>
FAX: 1-440-919-5086

**Comment by migetman9** [comment]
I don't believe we do. We don't even ask for gender or sex if I recall correctly. Just trying to be a good sport 😄

**Comment by earl7399** [comment]
Cool - and FWIW, I wasn't trying to be bitchy or whiny (*MIRACLE*, I know),
I just couldn't think of anywhere we'd use it. Asking for it and not using
it might end up feeling like a mean trick.

--
Robert Earl <r.a.earl@gmail.com>
FAX: 1-440-919-5086

**Comment by kitsuta** [comment]
This is tricky. Obviously if you got a suggestion for it, some people would want it. However, without knowing more about _why_ this was desired, it's hard to say if it should be done.

My initial reaction is that we should keep everything gender-agnostic in the system. Volunteers need to be trained to talk to and about people without using gendered words (many people think saying 'sir' and 'madam' is polite customer service, but they don't realize that it could just as easily misgender someone). Doing so is the best way to avoid customer service issues, either by honest mistake or by someone deciding that they really need to use the perceived-biological-sex of an attendee as their pronoun.

The trickiest part of this is when referring to someone in the third person while they're there, like when describing a problem they're having to a manager. Many people really, really want to say 'gentleman' or 'lady' when doing this because most other terms are pretty impolite (can you imagine saying "this person needs XYZ" to a manager? Yikes!). What we train people to do instead is a matter of opinion, although I personally would go with "This is [name if you remember it], they need XYZ."

In any case, I don't really thinking having their preferred gender recorded in the system will help these cases. These kinds of interactions are done 'off the grid' and it's not like someone would or even could reliably peek at an attendee's preferred gender before bringing them to the manager line.

Long story short: singular they is good. Singular they is right. Always use the singular they.

**Comment by earl7399** [comment]
​I also think that using a person's preferred first (or first and last)
name is the best option (esp. talking to that person), followed closely by
"they"​ if talking about them, whether or not they are present. It's a good
habit to get into.

--
Robert Earl <r.a.earl@gmail.com>
FAX: 1-440-919-5086
